.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @thomastaylor312 @technosophos @bacongobbler
```

Comment:
Chatting with @bacongobbler, he might not be necessarily interested in being a maintainer.
Feel free to add me instead, as I plan to allocate some more time to work here.

Reply:
Ok, I'll switch that out